docs/changelog.md

```diff
@@ -12,6 +12,13 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
+* **service-glean**
+  * ⚠️ **This is a breaking change**: Glean's configuration now requires an http client. Users need to pass one at construction.
```

Comment:
We should also update the glean sdk docs to mention this ...

Reply:
Will do https://bugzilla.mozilla.org/show_bug.cgi?id=1636951